e2e/tests/api-product-list.spec.ts

```
@@ -43,7 +43,7 @@ test.describe('APIProduct List Page - Display and Filters', () => {
     await expect(page.locator('th button div span:has-text("Created")')).toBeVisible();
   });
 
-  test('displays API Products from test fixtures', { tag: '@nightly' }, async ({ page }) => {
+  test('displays API Products from test fixtures', { tag: ['@nightly', '@api-management'] }, async ({ page }) => {
```

ESLint

[error] Replace `'displays·API·Products·from·test·fixtures',·{·tag:·['`@nightly`',·'`@api-management`']·},` with `⏎····'displays·API·Products·from·test·fixtures',⏎····{·tag:·['`@nightly`',·'`@api-management`']·},⏎···`

(prettier/prettier)